CoreFeatures/Database-postgresql/PostgresSchema/src/main/java/info/smart_tools/smartactors/database_postgresql/postgres_schema/search/Conditions.java

```diff
@@ -34,7 +34,7 @@ private Conditions(){
     public static void writeAndCondition(
             final QueryStatement query,
             final QueryWriterResolver resolver,
-            final FieldPath contextFieldPath,
+            final String contextFieldPath,
```

Comment:
Here I see many changes on method interface (FieldPath -> String). And I can think out the reason for this, but please clarify this reason. Did you check and make sure that this change does not affect any of past or current projects? (regular way of changing interfaces requires rise of version in 2nd digit)

Reply:
Old version parses field path immediately and stores it in contextFieldPath. With typecast we have an input like "fieldName": { "type": "decimal", ... }, so we cannot determine the field's type with field name. I decided to postpone field name parsing until we have information about its type.